apps/src/code-studio/pd/workshop_dashboard/components/survey_results/single_choice_responses.jsx

```diff
@@ -21,7 +22,16 @@ export default class SingleChoiceResponses extends React.Component {
         );
```

Comment:
similarly, should we call this method getTotalResponses now? I know the facilitator path is doing something different currently, but at least the well-understood part would be accurate and we can leave a comment about the facilitator path.